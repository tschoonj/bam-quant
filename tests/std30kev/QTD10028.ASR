File: C:\xrfdaten\2014KW30\chris\std30kev\std10028.spe
Start: 22.07.2014 10:05:13
Stop: 22.07.2014 10:06:53
BESSY: 298.9974637 297.9820803
IONI: 2.008953E-11 1.992611E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 97977
Run: 100000
Dead: 2
ICR: 170
OCR: 0
Accumulated BESSY: 59697.8439483
Accumulated Ioni: 3.97675072E-09
N Accumulation: 200
Normfactor:  5.1330736e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ni
exp_x 92.7203979492188 92.7204
exp_y 122.465805053711 122.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  28   1      7.472       852.        30.       1.07
