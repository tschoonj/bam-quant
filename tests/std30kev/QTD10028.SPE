$DATA:
0 2000
22917
14775
6382
1951
365
61
4
0
0
0
0
0
0
0
0
0
2
9
17
9
12
9
8
6
12
13
15
10
8
15
16
5
14
10
12
9
12
5
5
6
3
4
6
8
12
11
9
9
7
7
9
8
8
9
7
5
10
9
6
6
8
6
6
12
12
9
5
4
5
5
9
6
3
4
0
4
4
7
8
7
6
7
7
6
4
8
5
7
6
4
6
6
6
12
6
10
9
7
7
7
8
6
5
7
5
1
11
4
3
6
5
7
7
2
5
5
8
6
4
7
6
9
5
3
5
10
7
5
8
8
8
8
10
8
5
5
5
4
3
10
3
5
10
3
6
6
8
3
4
4
8
7
5
8
9
9
5
4
5
4
8
3
6
5
4
9
8
5
3
8
5
5
4
4
5
12
4
6
5
3
10
5
7
4
5
4
5
3
1
7
5
4
5
3
3
6
2
6
3
4
4
3
5
4
3
2
3
8
5
3
5
2
2
2
5
1
1
1
2
2
3
1
2
2
3
2
4
0
3
2
1
2
3
3
1
1
1
2
1
2
3
2
0
4
3
2
2
1
1
0
1
2
0
3
1
2
1
1
3
5
0
1
0
2
5
2
1
5
1
2
2
1
4
0
1
2
0
0
1
1
4
0
5
1
0
0
0
3
1
3
0
1
0
2
1
1
0
0
2
2
1
0
1
1
1
2
1
1
2
1
2
1
1
5
2
2
1
2
4
3
4
3
4
1
1
1
1
1
0
1
0
3
1
1
0
0
1
1
2
0
2
2
1
2
1
3
2
0
1
1
3
1
1
2
2
0
1
1
4
0
2
1
11
17
21
28
45
59
91
105
114
95
99
58
59
44
21
10
7
3
7
2
2
1
1
1
2
1
4
1
3
1
2
4
3
2
3
3
4
2
8
11
26
37
50
74
109
117
128
156
185
146
116
89
56
21
17
13
3
1
2
2
0
0
1
0
2
2
1
1
1
0
0
0
0
0
0
1
1
0
0
0
2
0
2
0
0
0
0
0
1
0
0
0
0
1
0
1
1
2
1
0
1
0
0
0
1
1
1
0
0
0
0
0
1
1
0
1
0
0
0
0
1
0
0
1
1
0
0
1
4
1
1
2
0
1
0
0
2
2
1
1
0
2
0
0
2
0
2
2
2
0
1
0
1
0
1
0
0
1
0
2
0
0
2
0
1
2
0
1
2
1
0
2
1
1
2
3
0
0
1
3
1
0
0
3
0
1
0
2
1
1
1
1
4
2
0
2
1
4
0
0
3
2
0
4
4
1
1
0
0
1
2
2
4
2
1
1
3
3
3
1
1
6
1
3
1
1
1
1
1
4
3
0
1
1
1
2
1
3
1
2
3
3
1
2
1
0
3
1
2
6
1
4
1
0
3
1
1
2
3
6
1
1
3
4
1
4
5
3
6
3
3
3
3
5
2
3
7
1
2
1
3
4
5
1
7
0
1
2
1
4
3
2
1
1
1
2
2
3
9
2
4
2
2
3
4
3
2
0
4
1
4
3
3
5
2
2
2
3
2
5
5
2
3
2
4
1
5
7
4
0
2
0
5
7
3
3
4
0
3
1
0
5
1
7
5
2
2
8
2
9
4
6
7
4
4
6
7
6
4
3
4
2
0
5
1
4
6
5
6
2
1
3
0
2
7
3
3
5
5
5
0
5
7
3
2
5
4
2
6
5
7
5
7
1
6
3
9
6
5
3
1
3
6
4
5
8
9
10
21
20
28
34
42
50
58
67
58
79
67
88
58
61
41
46
30
33
20
11
12
12
12
5
3
8
8
4
0
6
4
4
2
6
8
7
6
4
4
2
3
3
2
5
5
2
2
7
4
9
2
4
2
6
1
2
1
2
5
2
5
2
6
6
5
9
5
4
3
5
5
2
6
4
9
3
1
6
1
2
5
4
6
3
2
4
6
5
9
6
5
2
7
9
5
9
9
4
9
11
13
11
11
11
16
16
21
14
17
12
11
11
9
7
3
6
3
3
6
5
3
6
6
5
3
5
2
3
5
5
3
4
3
7
4
3
5
2
3
4
3
5
2
5
5
2
4
3
3
3
5
4
5
5
6
3
3
3
6
4
5
4
5
1
3
3
6
4
6
6
4
0
4
2
3
3
4
3
3
3
4
5
4
1
3
3
9
0
3
4
2
2
1
2
2
3
4
4
3
2
5
7
2
2
6
2
5
7
8
10
2
6
6
1
4
5
4
2
3
2
3
4
2
2
6
2
2
1
2
3
6
1
3
5
3
3
4
0
10
6
7
6
3
2
3
4
1
4
3
2
3
1
2
3
3
4
2
1
4
1
2
0
2
4
6
5
5
5
5
7
4
1
3
5
7
3
7
2
4
2
7
1
4
3
1
3
4
4
2
7
3
4
3
2
5
2
0
3
5
3
1
1
3
0
3
4
2
1
2
2
4
5
3
2
1
4
4
1
1
6
1
2
2
1
1
3
2
1
2
4
1
1
3
2
1
3
3
2
2
3
6
2
3
2
2
3
2
1
1
0
3
3
3
2
1
2
2
3
1
3
4
4
4
2
0
0
1
4
0
1
3
2
5
0
4
2
2
1
1
1
2
6
3
2
2
3
2
2
5
2
4
1
1
3
0
6
1
0
2
1
1
4
3
0
0
2
0
8
4
1
1
4
3
1
0
0
0
0
3
0
2
3
4
3
3
3
2
3
1
2
3
2
1
3
2
0
0
2
1
2
2
1
2
2
3
5
1
4
2
3
1
3
2
3
4
0
1
0
5
1
5
2
3
5
4
5
11
3
4
4
8
9
12
7
11
3
7
6
14
15
10
6
11
6
9
9
2
7
5
2
2
4
6
2
1
3
2
4
1
0
3
2
5
0
1
1
0
0
1
4
1
3
2
0
3
4
3
3
1
8
1
2
5
4
1
2
4
1
2
2
3
3
2
5
4
1
3
2
2
0
1
2
1
1
2
1
3
6
5
3
0
1
1
5
3
4
2
3
2
0
2
0
4
1
1
2
0
2
2
2
0
2
3
1
2
1
1
3
2
3
6
7
4
0
0
5
2
3
5
1
0
5
5
2
2
1
1
2
3
2
4
4
3
4
2
4
6
3
0
3
1
1
1
1
1
4
0
2
2
1
3
1
3
1
4
6
1
3
3
4
3
7
4
4
4
1
6
8
1
7
6
5
7
2
4
1
3
2
4
5
2
2
4
4
2
1
4
1
7
2
3
3
2
4
1
3
3
4
4
3
7
2
2
4
3
2
2
2
5
6
2
2
8
0
1
4
5
7
3
5
3
1
4
2
2
7
4
7
11
4
4
1
6
3
7
3
4
4
7
3
2
5
1
5
5
3
3
4
3
5
6
4
6
3
2
5
8
4
9
7
6
6
4
3
4
7
4
4
6
13
7
7
8
8
4
6
8
6
9
7
8
11
8
7
17
9
5
10
4
12
6
7
11
10
9
13
11
8
11
11
9
8
12
12
12
8
15
10
10
9
10
11
13
11
14
16
12
11
10
10
14
9
10
18
13
17
15
17
16
19
15
8
22
21
10
26
17
18
21
20
20
18
20
14
24
19
15
24
21
23
21
24
23
17
22
22
17
28
23
17
28
37
28
22
18
19
37
23
27
36
33
36
41
27
28
34
40
22
36
29
41
37
37
32
33
32
31
36
42
28
39
29
25
37
38
37
32
28
33
38
30
35
47
33
32
32
22
29
34
23
25
25
21
25
29
33
27
25
25
24
42
26
29
24
18
26
30
28
27
26
17
15
22
22
27
23
16
24
20
31
26
19
26
22
18
20
25
31
27
22
23
22
12
16
25
25
23
23
18
18
17
22
24
19
26
24
19
20
28
22
18
27
17
21
19
23
22
22
34
30
33
30
41
33
37
34
49
43
46
51
54
55
71
61
54
64
61
69
55
50
63
49
54
56
49
59
62
44
43
49
35
38
37
47
39
37
22
26
25
22
18
23
15
14
14
10
5
1
3
1
2
1
2
1
2
2
2
0
0
0
0
0
0
1
1
0
0
1
1
0
1
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10028.spe
Start: 22.07.2014 10:05:13
Stop: 22.07.2014 10:06:53
BESSY: 298.9974637 297.9820803
IONI: 2.008953E-11 1.992611E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 97977
Run: 100000
Dead: 2
ICR: 170
OCR: 0
Accumulated BESSY: 59697.8439483
Accumulated Ioni: 3.97675072E-09
N Accumulation: 200
Normfactor:  5.1330736e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Ni
 
exp_x 92.7203979492188 92.7204
exp_y 122.465805053711 122.4658
exp_z 53.8899993896484 53.8901
exp_w 135 135.0001
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  28   1      7.472       852.        30.       1.07
