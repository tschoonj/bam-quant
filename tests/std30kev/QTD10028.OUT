     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:47  บ   
     บ  Spectrum: QTD10028.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        6 iterations done    บ   
     บ  ChiSquare =     5.2   last change =   .06%    lambda= 1.E-04   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -94.5 ๑   100.0   บ     -94.5 ๑    22.1   บ   
     บ  GAIN (eV/ch)  บ     20.388 ๑   2.000   บ    20.388 ๑    .057   บ   
     บ det NOISE (eV) บ      124.7 ๑    40.0   บ     124.7 ๑    40.6   บ   
     บ  FANO factor   บ       .052 ๑    .050   บ      .052 ๑    .042   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Ni-Ka                               852. ๑     30.           บ   
     บ    KA1         7.478    .66126         563. ๑     20.      1.1  บ   
     บ              371.420    154.57          15.           9.47E-01  บ   
     บ    KA2         7.461    .33874         289. ๑     11.      1.3  บ   
     บ              370.586    154.51          15.           9.47E-01  บ   
     บ                                                                 บ   
     บ  2 Ni-Kb                              1316. ๑     37.           บ   
     บ    KB1         8.265   1.00000        1316. ๑     37.      2.1  บ   
     บ              410.021    157.39          14.           9.60E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
